app/assets/javascripts/lookup_keys.js

```diff
@@ -280,14 +280,16 @@ function build_match() {
 
 function toggle_lookupkey_hidden(checkbox) {
   var default_value = $('#'+checkbox.id.replace(/hidden_value$/, "default_value"));
-  var lookup_values = $(checkbox.closest('.fields')).find('.lookup_values>table [id$="value"]');
-  toggle_value_hidden(default_value, checkbox.checked);
+  var fields = $(checkbox.closest('.fields'));
```

Comment:
no need for this variable, or for the `$()` wrapper (`.closest()` returns a jQuery object already)